Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -1311,6 +1311,19 @@
     interactSuccessString: petting-success-tarantula
     interactFailureString: petting-failure-generic
   - type: Puller
+  - type: MobState
+    thresholds:
+      0: Alive
+      75: Critical
+      125: Dead
+  - type: MeleeWeapon
+    hidden: true
+    angle: 0
+    animation: WeaponArcBite
+    damage:
+      types:
+        Piercing: 15
+        Poison: 7
```

Comment:
This is exceptionally strong. It would also be better if they injected poison rather than just doing poison damage. Look at how Space Ticks do this.

Reply:
> This is exceptionally strong. It would also be better if they injected poison rather than just doing poison damage. Look at how Space Ticks do this.

I edit it.